transport-native-unix-common/src/main/c/netty_unix_errors.c

```diff
@@ -37,42 +37,42 @@ static char* exceptionMessage(char* msg, int error) {
 }
 
 // Exported C methods
```

Comment:
Can we clarify the terminology here? They are no longer exported from a "C" perspective ... they are more now strictly "Exported JNI methods, but not exported C methods".

Reply:
See https://github.com/netty/netty/pull/8207#discussion_r211333017